<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibrate</class>
 <widget class="QDialog" name="Calibrate">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration details</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="3">
    <widget class="QSpinBox" name="CalibrationDelayAfterRestartInHours">
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="CalibrationOffsetSamplesFromCurrentTime">
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="checkSyncAxis">
     <property name="text">
      <string>Synchronize Y axis (-sa)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="runCalibration">
     <property name="text">
      <string>Run calibration process (-fmc)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Calibration Delay After Restart, Hours (-cad)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="NumberOfSamplesForCalibration">
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Calibration Sample Periodicity, Minutes (-caspm)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Number Of Samples For Calibration (-cans)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="MinimumFlowToSample">
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QCheckBox" name="checkSync">
     <property name="text">
      <string>Synchronze start point with pipe start (-si)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="onSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="onCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="checkOpt">
     <property name="text">
      <string>Calculate optimal value (-opt)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="CalibrationSamplePeriodicityInMinutes">
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="CalibrationPeriodicityInHours">
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Calibration Sample Periodicity, Hours (-casph)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkLSQ">
     <property name="text">
      <string>Apply LSQ (-lsq)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Minimum Flow To Sample (-camf)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Calibration Offset Samples From Current Time (-cao)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Save trend to CSV file (-co)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="outputFile"/>
   </item>
   <item row="7" column="3">
    <widget class="QPushButton" name="selectFile">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
